src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -521,6 +520,18 @@ def update_os_patch_configuration_sub_setting(self, patch_configuration_sub_sett
             raise
     # endregion
 
+    # region Reboot Management
+    def is_reboot_pending(self):
+        """ Checks if there is a pending reboot on the machine. """
+        try:
+            pending_file_exists = os.path.isfile(self.REBOOT_PENDING_FILE_PATH)
```

Comment:
Do we not need: pending_processes_exists = self.package_manager.do_processes_require_restart() here?

Reply:
No, it's only functionally used in the other package managers.